platforms/sdl/base/AppPlatform_sdl_base.hpp

```diff
@@ -39,7 +39,15 @@ class AppPlatform_sdl_base : public AppPlatform
 	bool shiftPressed() override;
 	void setShiftPressed(bool b, bool isLeft);
 	
-	bool isTouchscreen() override { return false; }
+	bool isTouchscreen() override
```

Comment:
This ought to be touched by @TheBrokenRail in his Android SDL port. Undo this change.

Reply:
I checked his PR so yeah, undo this.